easybuild/main.py

```diff
@@ -266,7 +265,7 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
         # TODO add general_options (i.e. robot) to build options
         orig_paths, general_options = parse_easystack(options.easystack)
```

Comment:
`general_options` is not correct here, but this is being fixed in #4057 (renamed to `opts_per_ec`)